src/pages/SearchPage.js

```diff
@@ -215,5 +231,12 @@ export default compose(
         betas: {
             key: ONYXKEYS.BETAS,
         },
+        isSearchingForReports: {
+            key: ONYXKEYS.IS_SEARCHING_FOR_REPORTS,
+            initWithStoredValues: false,
+        },
+        network: {
+            key: ONYXKEYS.NETWORK,
```

Comment:
I think we should use the `withNetwork` HOC instead of `withOnyx`. Also `ONYXKEYS.NETWORK` should typically be `initWithStoredValues: false`, because we don't have any reason to think that the persisted value for `isOffline` should be accurate the next time the user opens the app.

Reply:
This is a class component still.

Reply:
Updating to withNetwork since it will reduce a subscription.

But there are so many places where we do not consider this...

>we don't have any reason to think that the persisted value for isOffline should be accurate the next time the user opens the app

nothing super bad happens AFAICT